themes/custom/essex/css/preliminary.css

```diff
@@ -79,18 +81,19 @@
     display: block;
   }
 }
+
 /* End */
 
 /* Added for https://nomensa.atlassian.net/browse/ECCI-471 */
-.field--name-localgov-subsites-content > .field__item:first-child {
+.field--name-localgov-subsites-content>.field__item:first-child {
```

Comment:
Is this a deliberate change in formatting (to remove spacing around _>_) ?  Will they get reverted by someone else's IDE?